site/documentation/content/admin-guide/monitoring-tracing-config.md

```diff
@@ -98,22 +98,31 @@ individual entries in the log with each other and thus *trace* the processing of
 However, this is usually a very tedious (and error prone) process and the relevant information is often only logged at
 a level (e.g. *DEBUG*) that is not used in production (often *INFO* or above).
 
-In order to address this problem, Hono's service components are instrumented using [OpenTracing](https://opentracing.io/).
-OpenTracing provides *Vendor-neutral APIs and instrumentation for distributed tracing*. The Hono container images all
-include the [Jaeger Java client](https://www.jaegertracing.io/docs/1.32/client-libraries/) and can be configured to
-send tracing information to a [Jaeger collector](https://www.jaegertracing.io/docs/1.32/architecture/).
-Please refer to the
-[Jaeger client documentation](https://github.com/jaegertracing/jaeger-client-java/blob/master/jaeger-core/README.md)
-for details regarding configuration.
+In order to address this problem, Hono's service components are instrumented using [OpenTelemetry](https://opentelemetry.io/).
```

Comment:
FMPOV this is not correct as we are still using OpenTracing for instrumenting the code. What we have changed is the OpenTracing implementation: Jaeger -> OpenTelemetry. or am I mistaken?

Reply:
Looking at the component as a whole and at its trace output as well as its configuration, I think the focus is here on OpenTelemetry now.
I've adapted the documentation here a bit.